apps/admin-gui/src/app/shared/components/entityless-attribute-keys-list/entityless-attribute-keys-list.component.ts

```diff
@@ -46,7 +46,7 @@ export class EntitylessAttributeKeysListComponent implements OnChanges, OnInit {
     this.setDataSource();
   }
 
-  @ViewChild(MatPaginator) paginator: MatPaginator;
+  @ViewChild(TableWrapperComponent, {static: true}) child: TableWrapperComponent;
```

Comment:
![image](https://user-images.githubusercontent.com/32871167/122238683-a0b21480-cec0-11eb-9e02-4a7d2f9e29af.png)
To get to this do: perun admin -> attributes -> click on some entityless attribute definition -> in dialog click on show keys(left down)